frontend/device/gesturedetector.lua

```diff
@@ -144,6 +147,15 @@ end
 --[[
 tap2 is the later tap
 --]]
+function GestureDetector:isTapBounce(tap1, tap2)
+    local tv_diff = tap2.timev - tap1.timev
+    return (
+        math.abs(tap1.x - tap2.x) < self.SINGLE_TAP_BOUNCE_DISTANCE and
```

Comment:
Taking an absolute value looks a bit suspicious at a glance? Could you explain the rationale?

Reply:
Not my rationale. This is a copy of isDoubleTap() just after - just to use different constants.

But taking the absolute value looks right to me when we want a distance - and make sure the 2 taps (whether bounce or real double tap) are near each others - so the user is not taping with 2 fingers far from each other.
So, 2 taps far from each other is not considered a bounce (or, it's an ultra wide bounce :) and probably not an accident :)
(I guess this might also be to not catch two-fingers tap - but not certain as 2 fingers tap must use 2 different slots.)

Reply:
I mean suspicious more in the sense that abs(3-4) = 1, for example. But I don't recall otoh how all this works.

Reply:
@Frenzie This one is alright, what it does is check if the difference in x/y coordinates between taps is in a square of length `self.SINGLE_TAP_BOUNCE_DISTANCE`.